modules/table/src/main/java/org/apache/ignite/internal/table/distributed/SortedIndexLocker.java

```diff
@@ -95,39 +96,54 @@ public CompletableFuture<Void> locksForLookup(UUID txId, BinaryRow tableRow) {
      * @return A future representing a result.
```

Comment:
there is no description, what is the result. Same for `acquireLockNextKey`.

Reply:
Added description.